docs/internals/optimizer.rst

```diff
@@ -26,6 +26,10 @@ One can use ``solc --ir-optimized --optimize`` to produce an
 optimized Yul IR for a Solidity source. Similarly, one can use ``solc --strict-assembly --optimize``
 for a stand-alone Yul mode.
 
+.. note::
+    The PeepholeOptimizer is always enabled by default and can only be turned off via the
```

Comment:
Would it make sense to mention what the peephole optimizer is, since I don't think we have it documented anywhere?

Reply:
From what I gather, `Peephole` refers to a common optimization method, so I guess that's why it does not have its own doc section. Maybe we could add an external link ([wikipedia article](https://en.wikipedia.org/wiki/Peephole_optimization)) as a reference to the technique itself.

Reply:
```suggestion
    The `PeepholeOptimizer <https://en.wikipedia.org/wiki/Peephole_optimization>` is always enabled by default and can only be turned off via the
```